docs/sections/user_guide/cli/tools/rocoto/realize-exec-stdout-verbose.out

```diff
@@ -1,21 +1,21 @@
-[2024-05-23T19:39:16]    DEBUG Command: uw rocoto realize --config-file rocoto.yaml --verbose
-[2024-05-23T19:39:16]    DEBUG Dereferencing, current value:
-[2024-05-23T19:39:16]    DEBUG   workflow:
-[2024-05-23T19:39:16]    DEBUG     attrs:
-[2024-05-23T19:39:16]    DEBUG       realtime: false
-[2024-05-23T19:39:16]    DEBUG       scheduler: slurm
-[2024-05-23T19:39:16]    DEBUG     cycledef:
-[2024-05-23T19:39:16]    DEBUG     - attrs:
-[2024-05-23T19:39:16]    DEBUG         group: howdy
-[2024-05-23T19:39:16]    DEBUG       spec: 202209290000 202209300000 06:00:00
+[2024-07-09T00:31:39]    DEBUG Command: uw rocoto realize --config-file rocoto.yaml --verbose
+[2024-07-09T00:31:39]    DEBUG Dereferencing, current value:
+[2024-07-09T00:31:39]    DEBUG   workflow:
+[2024-07-09T00:31:39]    DEBUG     attrs:
+[2024-07-09T00:31:39]    DEBUG       realtime: false
+[2024-07-09T00:31:39]    DEBUG       scheduler: slurm
+[2024-07-09T00:31:39]    DEBUG     cycledef:
+[2024-07-09T00:31:39]    DEBUG     - attrs:
+[2024-07-09T00:31:39]    DEBUG         group: howdy
+[2024-07-09T00:31:39]    DEBUG       spec: 202209290000 202209300000 06:00:00
 ...
-[2024-05-23T19:39:16]    DEBUG   account: '&ACCOUNT;'
-[2024-05-23T19:39:16]    DEBUG   attrs:
-[2024-05-23T19:39:16]    DEBUG     cycledefs: howdy
-[2024-05-23T19:39:16]    DEBUG   command: echo hello $person
-[2024-05-23T19:39:16]    DEBUG   envars:
-[2024-05-23T19:39:16]    DEBUG     person: siri
-[2024-05-23T19:39:16]    DEBUG   jobname: hello
-[2024-05-23T19:39:16]    DEBUG   nodes: 1:ppn=1
-[2024-05-23T19:39:16]    DEBUG   walltime: 00:01:00
-[2024-05-23T19:39:16]     INFO 0 Rocoto validation errors found
+[2024-07-09T00:31:39]    DEBUG   attrs:
+[2024-07-09T00:31:39]    DEBUG     cycledefs: howdy
+[2024-07-09T00:31:39]    DEBUG   account: '&ACCOUNT;'
+[2024-07-09T00:31:39]    DEBUG   command: echo hello $person
+[2024-07-09T00:31:39]    DEBUG   jobname: hello
+[2024-07-09T00:31:39]    DEBUG   nodes: 1:ppn=1
+[2024-07-09T00:31:39]    DEBUG   walltime: 00:01:00
+[2024-07-09T00:31:39]    DEBUG   envars:
+[2024-07-09T00:31:39]    DEBUG     person: siri
+[2024-07-09T00:31:39]     INFO 0 Rocoto validation errors found
```

Comment:
The debug output from dereferencing now uses the new, shared `yaml_to_str()` function (see below), which preserves the order of keys from the input config, which accounts for the reordered output.